client/components/material-icon/index.jsx

```diff
@@ -0,0 +1,52 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import svg4everybody from 'svg4everybody';
+
+const isBrowser = typeof window !== 'undefined';
+if ( isBrowser ) {
```

Comment:
@sgomes @griffbrad do we want this at every usage or should we move this to `lib/polyfill`?

Reply:
I'd favor adding it to client/boot/polyfills.

Reply:
Made this switch in b89d362.  Seems to fit in better with the way we handle other polyfills.

Reply:
Looks good to me 👍